### PR TITLE
Added publication

### PR DESCRIPTION
**Pull request by danielhstahl**
I wanted to write to a publication.  I added the functionality.  This is optional and can be called using
`--publication myPublication`

If there are no publications, it throws an error.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// from https://github.com/jxnblk/writing/blob/gh-pages/medium.js
```
Comment (RichardLitt) [inline index.js]: Why was this removed?

File: index.js [hunk 2]
```diff
@@ -41,9 +41,7 @@ function main (options, done) {
 
   let content = `
   # ${title}
-
   ${matter.body}
-
```
Comment (RichardLitt) [inline index.js]: Why were these removed?

File: index.js [hunk 3]
```diff
@@ -55,25 +53,39 @@ function main (options, done) {
...
+        userId: user.id,
+        title,
+        tags,
+        content,
+        canonicalUrl,
+        contentFormat: 'markdown',
+        publishStatus: 'draft'
+    };
+    const successMsg=`Draft post "${title}" published to Medium.com`.green;
+    publication?client.getPublicationsForUser({userId:user.id}, (err, publications)=>{
```
Comment (RichardLitt) [inline index.js]: Using a ternary here is a bit confusing. Could you break it into an if block for readability?

**Review by RichardLitt — COMMENTED** [review]
Overall, looks OK. Is there a way for us to test this short of creating a publication? 

You use a pretty different syntax from @yoshuawuyts. Perhaps reformat? I don't know if he cares, but it makes the code a bit confusing to read. You also remove some unnecessary things.

**Comment by danielhstahl** [comment]
I've addressed your comments.  That's a great question about testing...does Medium have a test API?  I don't see any unit tests in the original code.

**Comment by yoshuawuyts** [comment]
Oh dang, forgot to add standard to this repo for weird reasons. Ah well, we can fix formatting after a merge.

@RichardLitt added you as a collab on github and npm, feel free to merge and publish once you think things are fine :sparkles:

**Comment by RichardLitt** [comment]
Merged! I tested this by creating a publication. 

screen shot 2017-04-03 at 10 14 47 am

I also standardized the code, added `publication` to the documentation, and tested it a variety of other ways. Looks great to me. Than you! 

Pushed in 822cefb80de4c25e126f6633f106bc38d9fc853e.
